easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.14-GCC-7.2.0-2.29.eb

```diff
@@ -0,0 +1,18 @@
+name = 'Doxygen'
+version = '1.8.14'
+
+homepage = 'http://www.doxygen.org'
+description = """Doxygen is a documentation system for C++, C, Java, Objective-C, Python, 
+ IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D."""
+
+toolchain = {'name': 'GCC', 'version': '7.2.0-2.29'}
```

Comment:
@rubendibattista We usually use the `GCCcore` toolchain for something like `Doxygen`, so that the resulting module is compatible with both `foss` and `intel` (among other toolchains that use `GCCcore` as a base).